middleware/index.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import path from 'path'
+
+module.exports = ({middlewarePath='/falcor-postman', app}) => {
+  app.use(express.static(path.join(__dirname, '/../dist')))
+
+  return (req, res, next) => {
+    if (req.url === middlewarePath) {
+      res.status(200)
+      res.send(path.join(__dirname, '/../dist/fp.html'));
```

Comment:
let me try to understand, now the middleware serves a single html file compiled by `webpackServeBundle.js`? if yes why this file is in the `example` folder?